itou/files/management/commands/sync_s3_files.py

```diff
@@ -11,20 +13,54 @@ class Command(BaseCommand):
     # keeping individual query size manageable.
     BATCH_SIZE = 20_000
 
-    def handle(self, *args, **options):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--check-existing",
+            action=argparse.BooleanOptionalAction,
+            default=False,
+            help="Set to True to also check that all existing File objects reference existing bucket objects.",
+        )
+
+    def handle(self, *args, check_existing, **options):
         paginator = s3_client().get_paginator("list_objects_v2")
         page_iterator = paginator.paginate(Bucket=settings.AWS_STORAGE_BUCKET_NAME)
         batch = []
+        permanent_files_nb = 0
+        temporary_files_nb = 0
+        known_permanent_files_nb = 0
+        if check_existing:
+            known_keys = set(File.objects.values_list("key", flat=True))
+            self.logger.info("Checking existing files: %d file(s) in database before sync", len(known_keys))
```

Comment:
Si on en est là, on peut se douter qu’il y aura toujours plus d’un fichier en prod.
```suggestion
            self.logger.info("Checking existing files: %d files in database before sync", len(known_keys))
```